sql/core/src/test/resources/sql-tests/inputs/limit.sql

```diff
@@ -3,6 +3,7 @@
 SELECT * FROM testdata LIMIT 2;
 SELECT * FROM arraydata LIMIT 2;
 SELECT * FROM mapdata LIMIT 2;
+SELECT * FROM mapdata LIMIT 2 OFFSET 0;
```

Comment:
why testing OFFSET 0 not 2? why only test `mapdata`?

Reply:
If we think the pgsql tests are sufficient for OFFSET, we don't need to touch this file.

Reply:
OK